### DISCUSSION TITLE
Frontend Master can't verify my github student account.

### DISCUSSION THREAD

**Discussion by khoatran1602** [community/community#42318] [GitHub Education]
Hi everyone,

I already registered for Github student pack but when i apply for 6 month free courses on Frontend Master. They said that the api Github provided them returnt my account as not a student.

Is there any other way i can prove my identity?

**Comment by samus-aran** [comment]
Hi @khoatran1602 👋 

Are you still having issues applying to Frontend Masters course? Was [their support](https://frontendmasters.com/welcome/github-student-developers/) that stated that your student status wasn't verified?

**Reply by khoatran1602** [reply]
sorry, i though that no one noticed. They still not accept me. They say that GitHub gave them an api that get student's username or something and that link does not contain my username. But i have my account verified so i don't know what to do

**Comment by RONIE70** [comment]
> cuando solicité cursos gratuitos de 6 meses en Frontend Master. Dijeron que la api Github les proporcionó mi cuenta como no estudiante.
> 
> ¿Hay alguna otra forma en que pueda probar mi identidad?

You can get ahead in your career and access the best developer tools used in the industry, for free, with the GitHub Student Developer Pack (the Pack). The Pack is open to every student 13 years or older in a degree-granting program, and we’re committed to helping every student—no matter where you are. 

Most students apply for the Pack using their school-issued email address. However, students whose schools don’t provide email addresses are able to apply for the Pack using their dated student ID. But what do you do if you’d like to apply, but your school doesn’t provide student email addresses or IDs? We don’t want you to miss out, so we also accept alternative documentation as long as it reflects your current student status. 

[Get the GitHub Student Developer Pack ](https://education.github.com/pack)

What documentation is required to join the GitHub Student Developer Pack?
We want to make it as simple as possible for students anywhere to easily apply and go through the GitHub Student Developer Pack application process, while also keeping our commitment to sponsors to safeguard it from those who may take advantage of student offers. 

The better we make the Pack for students, the more we have to be aware of potential fraud. The current value of the collective offers in the Pack is over $12,000 per student. That’s a great deal for students and an attractive target for bad actors.

Who is eligible for the GitHub Student Developer Pack?
The Pack is available to any student, 13 years or older, in any degree-granting program that isn’t restricted by the institution’s accreditation—regardless of your location.

What proof do I need to submit for academic verification?
Submitting a school-issued email address is preferred because once we verify that the domain is associated with a qualifying institution and that the school’s email policies disallow non-student use of those emails, we can approve applicants using those domains faster.

If your school doesn’t issue email, or we do not consider your school’s email sufficient to demonstrate your current academic status, then we will ask for a different form of proof. This could be a dated student ID, a current registration document, a time-stamped screenshot of your school portal, a letter on school letterhead confirming you are a currently-registered student, etc.

How to apply for the GitHub Student Developer Pack
Visit [GitHub Education](https://education.github.com/) and click Get benefits in the top-right corner.
Under “Which best describes your academic status?”, select Student.
Then, complete one of the following:
If you have a school-issued email, select (or add it).
Please note that email addresses must be verified and any addresses with plus signs (“+”) are not eligible.
If you don’t have a school-issued email, follow the prompts to fill out additional information.You’re still eligible even if you only have a personal Gmail address, as long as you can provide alternative documentation to verify your current student status.
You may be prompted for more information. When prompted, upload an image that demonstrates your academic status or confirms your enrollment.

If you do not have a dated student ID to upload, you may upload a current transcript, letter on school letterhead, or any documentation that demonstrates your current enrollment. Make sure the image is clear and easy to read. Other languages are acceptable but may take longer to translate. The following are a few examples of valid dated documentation:  Current transcripts or a letter from your school with a date are acceptable forms of valid dated documentation.
Enter your school’s name, share how you will use GitHub, and click Submit. We try to review and process applications within two weeks of submission.
I added my school-issued email, but I don’t see it.
Did you remember to [verify it](https://help.github.com/en/articles/verifying-your-email-address)? Unverified email addresses will not appear. Once verified, refresh the application page for your email to appear.

As an anti-fraud measure, email addresses with plus signs are not accepted.

I tried my school-issued email, but I’m still being asked for additional proof.
To qualify as a sole form of academic verification, GitHub requires that school-issued email be provided exclusively to currently-registered students—not to everyone who has applied to the school, nor retained by alumni of the school. We also monitor trends in academic domain usage to weed out potentially compromised domains.

If your student email cannot be used as a sole form of proof, upload another form of supporting documentation.

My school does not provide student emails or IDs. What

**Comment by chmajidnaeem** [comment]
use a perfect and honest information for student account

**Reply by khoatran1602** [reply]
What do u mean by that? I am a student. Is this honest enough for u? If not, tell me. I provide more proof for u.
![Screenshot_20230316_010310_Brave](https://user-images.githubusercontent.com/61503687/225402217-9752013a-bf00-42e2-a6f9-9d06ea334e9e.jpg)

**Comment by Sowmya-149** [comment]
How many days it takes to accept the account?
